src/cryptojwt/serialize/item.py

```diff
@@ -0,0 +1,27 @@
+import json
```

Comment:
Unused code as well?

Reply:
Not really. I have a problem here and that is that all the storage stuff is really defined in oidcmsg. I don't want cryptojwt be dependent on oidcmsg (wrong way around) so there are no tests for storage usage in cryptojwt as it stands right now. Have contemplated adding something simple (shelve based for instance). 
Anyway this module should probably be moved to oidcmsg.

Reply:
I kept it because it's cryptojwt specific.